server/cert/server.crt

```diff
@@ -1,21 +1,17 @@
 -----BEGIN CERTIFICATE-----
-MIIDZjCCAk4CCQC4PbSJnokqvTANBgkqhkiG9w0BAQsFADB1MQswCQYDVQQGEwJu
-bDELMAkGA1UECAwCTkgxEjAQBgNVBAcMCUFtc3RlcmRhbTEbMBkGA1UECgwSR2Vt
-ZWVudGUgQW1zdGVyZGFtMREwDwYDVQQLDAhEYXRhcHVudDEVMBMGA1UEAwwMYW1z
-dGVyZGFtLm5sMB4XDTE5MDQxOTA5NTI0N1oXDTIwMDQxODA5NTI0N1owdTELMAkG
-A1UEBhMCbmwxCzAJBgNVBAgMAk5IMRIwEAYDVQQHDAlBbXN0ZXJkYW0xGzAZBgNV
-BAoMEkdlbWVlbnRlIEFtc3RlcmRhbTERMA8GA1UECwwIRGF0YXB1bnQxFTATBgNV
-BAMMDGFtc3RlcmRhbS5ubDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEB
-AMM78zKoX2aJG42ki+/eq95U2fpuVKDv+OAdyQpkMHboF52BKh00S7TY8NapVPCB
-3IxbYkPJasFAe5Ik8Eo06MtR2T2KODP5pDTwekdhW2hgtyx6H6UY9fuXH1kGqNOt
-8NyJP0tZ3kjPVlplG5ykgwCAjOtYFqYVpBHDC63jwsUoY+mfNrIUWacSztH9IkFU
-iYcFHDsGllxPyb4BqAW4SBRhqzuPUD3iNNLAX7VBDJed3EQAKNcTo51QwwOBxAQE
-oakNkUIbmswiB6St/d12RXnScV96Bz2wSD+LiYV5DRLkbZZE6iyTq1Za+07I85oQ
-Uu1a7ob1vumDnyRPJLPldwsCAwEAATANBgkqhkiG9w0BAQsFAAOCAQEAs9QDHsXc
-V/FVubufxDgdJXT8ikj6GsTPwfVHjTWmlF7ELmtVX6faJW6X5XDPsMEp87EA2cvk
-5Pw1gosgRF2V+G00pjwW+iCLXhjWrwQLfM14ZkKsl+rWb5k1naC5ZtJzeDJ8CyhY
-DA4b/g1y+d6spMtmTYdR6CXPP9sx0JAxSOMjpsN0/l1V4MRYGRiTPo9dwiLm/1j/
-2JGIHgofqbDAb9mqNEXxoJRy2yUEAj+k5VM8SpH96CGrjjH6ZHHFpHCcNDgDHdvU
-AjkCKcyUPpk5QZwVv7ynRW06cDbSd8HnAKNl8Pn2s94uOnEty6uUHrGFIEb3ptUP
-MDsdn6jULYCVKA==
+MIICwzCCAaugAwIBAgIJAPwOhhvAE7C3MA0GCSqGSIb3DQEBCwUAMBQxEjAQBgNV
+BAMTCWxvY2FsaG9zdDAeFw0yMDAzMDMxNTE5MzBaFw0zMDAzMDExNTE5MzBaMBQx
+EjAQBgNVBAMTCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
+ggEBAMR5s5WE0RHapTucx4o5o1ei2FcAoICdbUIqWEmNsxjm5UXRKlkCNfHClj43
+nS0dtNlmmOzm8zcm/B+9dPNNk+eX600asrFiecx7HgbiIUCfKhjc93Zitpj1fmp+
+gYSkYDHcIJbZZWkeTWklJmJPDs6BmUjaC3h2kB3ZjCoMY/WkG0UU0GLgcy5CVf/r
+lFTtkt1TW18P1fX0Pre8xTRclkQpwPbvMGhQl0WFJnbnRh6UtMj9aaVP3IPbrtSU
+DKjItMyJ4G6HLNcz5xXlc8VmvmnXueWMkZCA9oIUa6bF+xPHFLA9ofppjgmUUNII
+JeUZDjF5v3ss7OlbDHZAwveqltMCAwEAAaMYMBYwFAYDVR0RBA0wC4IJbG9jYWxo
+b3N0MA0GCSqGSIb3DQEBCwUAA4IBAQBMu8vgnbQCVXOOKL3BqCOwSDYy5YXRC5k/
+MKhG6NRpe9O/EiZOzWK7SW+tB6bhcdn1PaH2e8lMW0PTlBZW/XyNjPjLOyOBslWE
+hRUwjz0+cHTKXNgvbYoHE+zXnll4eZWhPaBF0ZBHA86TSfWHrCwlxjAWPd6IsabF
+ojnjCdokBZFCho0Z7e5fVaV+rPiVQ+49wadoT56CvLmmyIQpligFI0YGnUwckkkO
+u17F9hRp5nLelvtdOxKzRfj99gLSzIrtR1Y2Wxkfe21QYqIsi1WZhxnth+BjyHnn
+VPw9KrbunpaZ+ajd11n+24vwae7MvvzINHLbv7nJyHr8oKQKcAIS
```

Comment:
While this certificate is only used for 'testing' purposes, I still have `weird` feelings by publishing it.

An alternative would to generate the certificates when they are not there when you start the proxy.

Reply:
@janjaap what do you think about the proposal?

Reply:
Can you explain what feels 'weird' about having a self-signed certificate in this repo?